OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -114,14 +120,14 @@ public void Tick(IBot bot)
 				: baseBuilder.Info.StructureProductionInactiveDelay + randomFactor;
```

Comment:
Increasing this to `8 * world.LobbyInfo.GlobalSettings.OrderLatency` seems to fix the issue on fastest speed. The issue with duplicated build orders also occurs when the `BotProductionConsideration` actually disallows it (for example when the trait got disabled after the first build order was completed).